File: app/src/main/java/com/dmitrybrant/android/mandelbrot/MandelbrotView.java
package com.dmitrybrant.android.mandelbrot;

import android.util.AttributeSet;
import android.content.Context;

public class MandelbrotView extends MandelbrotViewBase {

    public MandelbrotView(Context context) {
        super(context);
        init();
    }

    public MandelbrotView(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public MandelbrotView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        init();
    }

    protected void init() {
        if (isInEditMode()) {
            return;
        }
        super.init(false);
    }
}

File: app/src/main/java/com/dmitrybrant/android/mandelbrot/MandelbrotView.kt
package com.dmitrybrant.android.mandelbrot

import android.content.Context
import android.util.AttributeSet

class MandelbrotView(context: Context, attrs: AttributeSet? = null) :
        MandelbrotViewBase(context, attrs) {
    init {
        super.setup(false)
    }
}
